source/components/UserIcon/styles.js

```diff
@@ -12,16 +12,18 @@ export default makeStyles({
     justifyContent: 'center',
     alignItems: 'center',
     position: 'relative',
+    fontSize: 34,
   },
   big: {
     width: 60,
     height: 60,
-    fontSize: 34,
   },
   medium: {
     width: 40,
     height: 40,
-    fontSize: 22,
+    '& > span': {
+      transform: 'scale(0.7)',
```

Comment:
why sacle it down?

Reply:
Sidebar account's emoji wasn't placed correctly